rdkit/Chem/UnitTestDescriptors.py

```diff
@@ -30,13 +30,17 @@ def feq(n1, n2, tol=1e-4):
 
 class TestCase(unittest.TestCase):
 
+  def tearDown(self):
+    RDLogger.EnableLog('rdApp.error')
+
   def testBadAtomHandling(self):
     smis = ('CC[Pu]', 'CC[*]')
     for smi in smis:
       m = Chem.MolFromSmiles(smi)
       self.assertTrue(m)
       for nm, fn in Descriptors._descList:
         try:
+          RDLogger.DisableLog('rdApp.error')
```

Comment:
Seems like this shouldn't be done inside the loop. Probably makes sense to move it up to the top of this method.